AutoMapper.AspNetCore.OData.EFCore/QueryableExtensions.cs

```diff
@@ -132,16 +134,25 @@ private static IQueryable<TModel> GetQueryable<TModel, TData>(this IQueryable<TD
         {
             
             var expansions = options.SelectExpand.GetExpansions(typeof(TModel));
+            
+            var includeProperties = expansions
+                .Select(list => new List<Expansion>(list))
+                .BuildIncludes<TModel>(options.SelectExpand.GetSelects());
+
+            var includePropertiesNotWorking = expansions
+                .Select(list => new List<Expansion>(list))
+                .BuildWithLiteralLists<TModel>(options.SelectExpand.GetSelects());
 
+            var includes = includeProperties
```

Comment:
@BlaiseD this part is a bit weird.
When I took the code from `LogicBuilder` ([MyPreviousComment](https://github.com/AutoMapper/AutoMapper.Extensions.OData/pull/218/files#r1807877719))
I got inheritance issue exactly as https://github.com/AutoMapper/AutoMapper.Extensions.OData/discussions/131, I debugged it, and found that `ProjectionBuilder.ProjectionRequest.ShouldExpand()` returns `false` only on inherited properties. (it compares two `MemberPath`s and because one of them's (reflection) reference is `BaseCoreBuilding` and not `CoreBuilding` it fails, which is make sense)

<img width="613" alt="image" src="https://github.com/user-attachments/assets/8206f587-4edc-4f9a-96c4-1097d7296a4b">

The weird part is, that without copying the code from LogicBuilder, this `MemberPath` is created without a reference to the base class and then it works, but without our literal-lists... so I merged those lists.....